.eslintrc.json

```diff
@@ -93,31 +94,12 @@
       "@typescript-eslint/parser": [".js", ".jsx", ".ts", ".tsx"]
     },
     "import/resolver": {
-      "node": {
-        "extensions": [".js", ".jsx", ".ts", ".tsx", ".json"],
-        "paths": ["src"]
-      },
-      "alias": {
-        "extensions": [".js", ".jsx", ".ts", ".tsx", ".json"],
-        "map": [
-          ["^jotai-urql$", "./src/index.ts"]
-        ]
+      "eslint-import-resolver-custom-alias": {
```

Comment:
There were issues resolving some modules (`vitest` to be precise) so this had to be used as older resolver is 4 years without any updates. 
As per: https://github.com/vitest-dev/vitest/issues/1827